physicsnemo/sym/eq/pde.py

```diff
@@ -77,7 +77,7 @@ def make_nodes(
         self,
         create_instances: int = 1,
         freeze_terms: Dict[str, List[int]] = {},
```

Comment:
Same comment here on `freeze_terms` about mutable default arguments, as with the list below (dictionaries are mutable)